src/domain/journey/subspace/pages/SubspaceProfile/SubspaceProfilePage.tsx

```diff
@@ -3,13 +3,16 @@ import React, { FC } from 'react';
 import { SettingsSection } from '@/domain/platform/admin/layout/EntitySettingsLayout/constants';
 import { SettingsPageProps } from '@/domain/platform/admin/layout/EntitySettingsLayout/types';
 import SubspaceProfileView from './SubspaceProfileView';
-import FormMode from '@/domain/platform/admin/components/FormMode';
 import SubspaceSettingsLayout from '@/domain/platform/admin/subspace/SubspaceSettingsLayout';
+import { useRouteResolver } from '@/main/routing/resolvers/RouteResolver';
 
 const ChallengeProfilePage: FC<SettingsPageProps> = ({ routePrefix = '../' }) => {
+  const { journeyPath } = useRouteResolver();
```

Comment:
@coderrabitai this is not true. `[][-1]` returns `undefined` in javascript. I am handling undefined. and journeyPath always has a value, at least `[]`

Reply:
@coderabbitai  I misspelled your name. I was saying, about your first comment that it is not true: 
`[][-1]` returns `undefined` in javascript. I am handling that `undefined` and the only thing that can happen is that `journeyPath` is itself undefined which never happens because of the implementaton in `useRouteResolver`,